src/commands/trialApp/removeTrialApp.ts

```diff
@@ -13,7 +14,13 @@ export async function removeTrialApp(context: IActionContext, node?: TrialAppTre
         node = await ext.tree.showTreeItemPicker<TrialAppTreeItem>(TrialAppTreeItem.contextValue, context);
     }
 
-    ext.context.globalState.update(TrialAppLoginSession, undefined);
-    delete ext.azureAccountTreeItem.trialAppNode;
-    await ext.tree.refresh();
+    const message: string = `Are you sure you want to remove Trial app "${node.client.fullName}"?`;
+    const remove: MessageItem = { title: 'Remove' };
```

Comment:
I'd prefer `ext.ui.showWarningMessage` instead of `window.showInformationMessage`. It handles 'cancels' better and a warning feels more appropriate